packages/components/psammead-episode-list/src/description.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+import { C_EBON } from '@bbc/psammead-styles/colours';
+
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const Description = styled.span`
+  ${({ service }) => getSansRegular(service)}
+  color: ${C_EBON};
+  display: block;
+  font-size: 15px;
```

Comment:
Hardcoded values for font size and line height will not work well with changing scripts, as different scripts expect different values for this

Additinally, use of `px` will not work with a11y users changing their font size

See psammead-paragraph, psammead-headings etc for examples of the functions we can use to set these values dynamically